archivessnake/findByRefid.py

```diff
@@ -22,139 +25,139 @@ def getAoTitle(ao):
         ancestor = client.get(ancestor_url).json()
         return ancestor["title"]
 
-def findDate(date):
+def findYear(date):
     if len(date) >= 4:
         if date[-4:].isdigit() and int(date[-4:]) >= 1850 and int(date[-4:]) <= 2020:
             year=date[-4:]
             return year
         else:
-            print(date)
             x=-5
             y=-1
             for r in range(len(date)):
                 if date[x:y].isdigit() and int(date[x:y]) >= 1850 and int(date[x:y]) <= 2020:
                     year=date[x:y]
-                    print(year)
                     return year
                     break
                 else:
                     x += -1
                     y += -1
-                
 
 def checkUndated(ao):
     if ao.get("dates")[0].get("expression") not in ["n.d.", "undated", "Undated"]:
         return True
     else:
         return False
 
+def findBulkDates(ao):
+    if ao.get("dates"):
+        if len(ao.get("dates")) == 2:
```

Comment:
It will--but when I'm calling it, if it returns false, I then grab the date. This is basically "check if there are multiple dates; if so, take bulk, not inclusive;" I'll add a comment.